src/http/ContentLengthInterpreter.h

```diff
@@ -9,6 +9,8 @@
 #ifndef SQUID_SRC_HTTP_CONTENTLENGTH_INTERPRETER_H
 #define SQUID_SRC_HTTP_CONTENTLENGTH_INTERPRETER_H
 
+#include "StatusCode.h"
```

Comment:
Please use the "http/" path prefix. We explicitly disable path searching into "-I.", it is an automake bug that sometimes these includes work.

Reply:
Done.